src/solvers/generic.ml

```diff
@@ -1,3 +1,5 @@
+(** Various old solvers. *)
```

Comment:
True?

Reply:
Well, most of this file is some old things like `DirtyBoxSolver`. I'm not sure what they exactly are.

Reply:
`LoadRunIncrSolver` is used for Gobview, and the `SolverStats` functor is used to get solver statistics. I'd argue these two are more important than `DirtyBoxSolver` and the other crap, so I think we should mention them in the summary.

Reply:
Ideally we wouldn't have such a generically named module as `Generic` (which seemingly has nothing to do with solvers) at all. `LoadRunIncrSolver` should be separate and `SolverStats` functor could be in the newly introduced `SolverStats` file module, but I won't refactor the code itself here, just expanded the synopsis.